config/config.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable sort-keys */
```

Comment:
Suggest not turning this off. The config object is reasonably large and would benefit from being alphabetised.

Reply:
I disagree, the config has a specific logical flow, app stuff, es stuff, analytics stuff. Removing it will make it harder to read.

Reply:
Ok, can those sections be added to the config so it is clear what relates to what.